src/Easify.Excel.ClosedXml.IntegrationTests/ExcelPropertyMapFactoryTests.cs

```diff
@@ -0,0 +1,228 @@
+// This software is part of the Easify.Ef Library
+// Copyright (C) 2018 Intermediate Capital Group
+// 
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+// 
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+// 
+
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Easify.Excel.ClosedXml.IntegrationTests.Helpers;
+using Easify.Excel.ClosedXml.IntegrationTests.Models;
+using Easify.Excel.Mapper;
+using Easify.Excel.Mapper.PropertyMap;
+using Easify.Excel.Mapper.PropertyMap.Convetions;
```

Comment:
Typo in the namespace Convetions

Reply:
Good one. That was a wide error in the code that I fixed in many places. It seems like I missed some 😄